src/Sprite/PaletteHeader.cpp

```diff
@@ -14,7 +14,7 @@ PaletteHeader PaletteHeader::CreatePaletteHeader()
 	PaletteHeader paletteHeader;
 	paletteHeader.remainingTagCount = 1;
 
-	const uint64_t dataSize = sizeof(Palette);
+	const uint64_t dataSize = sizeof(Palette8Bit);
```

Comment:
I wonder if this value should be passed in as a parameter. Could make it an optional parameter, with the current value as default.

Could also potentially set default field values rather than use a factory method to construct the instances. Though that idea doesn't play well with making the above value a parameter.

Is this really a `struct`? It seems like we're trying to maintain some sort of invariant here.

Probably a discussion for the future. Maybe better not to worry about it in this PR.